imbi/server.py

```diff
@@ -227,7 +227,9 @@ def load_configuration(config: str, debug: bool) -> typing.Tuple[dict, dict]:
         'project_configuration': {
             'ssm_prefix_template': project_configuration.get(
                 'ssm_prefix_template',
-                '/{namespace_slug}/{project_type_slug}/{project_slug}/')
+                '/{namespace_slug}/{project_type_slug}/{project_slug}/'),
+            'component_score_fact_id': project_configuration.get(
+                'component_score_fact_id')
```

Comment:
I don't think this should live under `project_configuration`, which is all in reference to the Configuration tab of a project.